Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -559,13 +603,26 @@ def enrich_dbot_and_display_url_analysis_results(intezer_result, intezer_api):
     presentable_result += f'[Analysis Link]({intezer_result["analysis_url"]})\n'
 
     downloaded_file_presentable_result = ''
+    file_entry: dict = {}
     if 'downloaded_file' in intezer_result:
         downloaded_file = intezer_result.pop('downloaded_file')
         presentable_result += f'Downloaded file SHA256: {downloaded_file["sha256"]}\n'
         presentable_result += f'Downloaded file Verdict: **{downloaded_file["analysis_summary"]["verdict_type"]}**\n'
         downloaded_file_analysis = FileAnalysis.from_analysis_id(downloaded_file['analysis_id'], intezer_api)
-        intezer_result['downloaded_file'] = downloaded_file_analysis.result()
-        downloaded_file_presentable_result = _file_analysis_presentable_code(downloaded_file_analysis.result())
+        download_file_result = downloaded_file_analysis.result()
+        intezer_result['downloaded_file'] = download_file_result
+        metadata = downloaded_file_analysis.get_root_analysis().metadata
```

Comment:
Shouldn't the metadata() function be called?
```suggestion
        metadata = downloaded_file_analysis.get_root_analysis().metadata()
```